src/Aspire.Dashboard/Components/Layout/MainLayout.razor.cs

```diff
@@ -158,35 +156,42 @@ public async Task LaunchSettingsAsync()
         _openPageDialog = await DialogService.ShowPanelAsync<SettingsDialog>(parameters).ConfigureAwait(true);
     }
 
-    public async Task OnPageKeyDownAsync(KeyboardEventArgs args)
+    public IReadOnlySet<AspireKeyboardShortcut> SubscribedShortcuts { get; } = new HashSet<AspireKeyboardShortcut>
     {
-        if (args.OnlyShiftPressed())
+        AspireKeyboardShortcut.Help,
+        AspireKeyboardShortcut.Settings,
+        AspireKeyboardShortcut.GoToResources,
+        AspireKeyboardShortcut.GoToConsoleLogs,
+        AspireKeyboardShortcut.GoToStructuredLogs,
+        AspireKeyboardShortcut.GoToTraces,
+        AspireKeyboardShortcut.GoToMetrics
+    };
+
+    public async Task OnPageKeyDownAsync(AspireKeyboardShortcut shortcut)
+    {
+        switch (shortcut)
```

Comment:
Can you add a log message here that a keyboard shortcut was received that includes the type.